resources/assets/coffee/react/beatmapset-page/header.coffee

```diff
@@ -142,15 +142,15 @@ export class Header extends React.Component
 
           el BeatmapsetMapping, beatmapset: @props.beatmapset
 
-          if currentUser.id? && @props.beatmapset.availability
+          if currentUser.id?
```

Comment:
`parts-removed` is incorrectly visible on normal maps.


disabled | url | description
---|---|---
false | empty | normal (no availability info box)
false | present | parts-removed
true | empty | disabled, no info link
true | present | disabled, with info link